lib/ticks-to-tree.js

```diff
@@ -4,8 +4,9 @@ const { spawnSync } = require('child_process')
 const { join } = require('path')
 const debug = require('debug')('0x: ticks-to-tree')
 const assert = require('assert')
+const escape = require('escape-string-regexp')
 
-const preloadDirRx = RegExp(join(__dirname, 'preload'))
+const preloadDirRx = RegExp(join(escape(__dirname), 'preload'))
```

Comment:
This won't escape `windows-path\preload`, but I don't think that's a problem… `escape(join())` would make sure the entire thing is escaped, regardless

Reply:
Actually escaping before joining does introduce more backslashes which _may_ be a problem for unicodey windows paths? I'm not really familiar with them.

Reply:
escaping after joining does *not* work. I tried that first.

Reply:
Ah, that's unintuitive. Alright :+1: